### PR TITLE
Updated CLI use examples in Documention

### PR DESCRIPTION
**Pull request by hotzenklotz**
### Description:
- Updated CLI use examples in the docs

### Issues:
- None

### Todos:
Make sure to delete unnecessary points or to check all before merging:
 - [ ] Updated Changelog
 - [x] Updated Documentation
 - [ ] Added / Updated Tests
 - [ ] Considered adding this to the Examples

### CODE REVIEW

File: docs/src/cli/index.md [hunk 1]
```diff
@@ -48,23 +48,23 @@ webknossos convert \
 
 # Create downsampled magnifications
 webknossos downsample data/target
-webknossos downsample --layer_name color data/target
+webknossos downsample --layer-name color data/target
 
 # Compress data in-place (mostly useful for segmentation)
-webknossos compress --layer_name segmentation data/target
+webknossos compress --layer-name segmentation data/target
```
Comment (hotzenklotz) [inline docs/src/cli/index.md:54]: @markbader According to typer `compress` has no `--layer-name` option. Yes or no?

File: docs/src/cli/index.md [hunk 2]
```diff
@@ -48,23 +48,23 @@ webknossos convert \
...
-webknossos convert-knossos --layer_name color data/source/mag1 data/target
+webknossos convert-knossos --layer-name color data/source/mag1 data/target
 
 # Convert NIFTI file to wkw file
-webknossos convert-nifti --layer-name color --scale 10,10,30 data/source/nifti_file data/target
+webknossos convert-nifti --layer-name color --voxel-size 10,10,30 data/source/nifti_file data/target
 
 # Convert folder with NIFTI files to wkw files
-webknossos convert-nifti --color-file one_nifti_file --segmentation_file --scale 10,10,30 another_nifti data/source/ data/target
+webknossos convert-nifti --color-file one_nifti_file --segmentation_file --voxel-size 10,10,30 another_nifti data/source/ data/target
```
Comment (hotzenklotz) [inline docs/src/cli/index.md:64]:
```suggestion
webknossos convert-nifti --color-file one_nifti_file --segmentation-file --voxel-size 10,10,30 another_nifti data/source/ data/target
```

File: docs/src/cli/index.md [hunk 3]
```diff
@@ -48,23 +48,23 @@ webknossos convert \
...
 
 # Convert RAW file to wkw file
-webknossos convert-raw --layer-name color --scale 10,10,30 --input-dtype uint8 --shape 2048,2048,1024 data/source/raw_file.raw data/target
+webknossos convert-raw --layer-name color --voxel-size 10,10,30 --input-dtype uint8 --shape 2048,2048,1024 data/source/raw_file.raw data/target
```
Comment (hotzenklotz) [inline docs/src/cli/index.md:67]:
```suggestion
webknossos convert-raw --layer-name color --voxel-size 10,10,30 --dtype uint8 --shape 2048,2048,1024 data/source/raw_file.raw data/target
```

**Comment by hotzenklotz** [comment]
@markbader Please double check if the CLI examples in the docs use the correct options?

File: docs/src/cli/index.md (hunk 1 above)
Comment (markbader) [reply docs/src/cli/index.md:54]: You are right, compress automatically compresses every not compressed file.

Comment (hotzenklotz) [inline docs/src/cli/index.md:54]:
```suggestion
webknossos compress data/target
```

File: docs/src/cli/index.md [hunk 4]
```diff
@@ -48,23 +48,23 @@ webknossos convert \
...
+webknossos downsample --layer-name color data/target
 
 # Compress data in-place (mostly useful for segmentation)
-webknossos compress --layer_name segmentation data/target
+webknossos compress --layer-name segmentation data/target
 webknossos compress data/target
 
 # Convert Knossos cubes to wkw cubes
-webknossos convert-knossos --layer_name color data/source/mag1 data/target
+webknossos convert-knossos --layer-name color data/source/mag1 data/target
```
Comment (markbader) [inline docs/src/cli/index.md:58]: The voxel-size option is mandatory here:
```suggestion
webknossos convert-knossos --layer-name color --voxel-size 11.24,11.24,25 data/source/mag1 data/target
```

File: docs/src/cli/index.md [hunk 5]
```diff
@@ -48,23 +48,23 @@ webknossos convert \
...
 
 # Convert NIFTI file to wkw file
-webknossos convert-nifti --layer-name color --scale 10,10,30 data/source/nifti_file data/target
+webknossos convert-nifti --layer-name color --voxel-size 10,10,30 data/source/nifti_file data/target
 
 # Convert folder with NIFTI files to wkw files
-webknossos convert-nifti --color-file one_nifti_file --segmentation_file --scale 10,10,30 another_nifti data/source/ data/target
+webknossos convert-nifti --color-file one_nifti_file --segmentation-file --voxel-size 10,10,30 another_nifti data/source/ data/target
```
Comment (markbader) [inline docs/src/cli/index.md:64]: One keyword argument is at the wrong position here:
```suggestion
webknossos convert-nifti --color-file one_nifti_file --segmentation-file another_nifti --voxel-size 10,10,30 data/source/ data/target
```

**Review by markbader — COMMENTED** [review]
There are two more small issues I found within the examples, except for that it looks good now.

**Review by markbader — APPROVED** [review]

**Pull request merged**